public/app/plugins/panel/histogram/migrations.ts

```diff
@@ -0,0 +1,30 @@
+import { PanelTypeChangedHandler } from '@grafana/data';
+
+/*
+ * This is called when the panel changes from another panel
+ */
+export const changeToHistogramPanelMigrationHandler: PanelTypeChangedHandler = (
+  panel,
+  prevPluginId,
+  prevOptions,
+  prevFieldConfig
+) => {
+  if (prevPluginId === 'graph') {
+    const graphOptions: GraphOptions = prevOptions.angular;
+
+    if (graphOptions.xaxis?.mode === 'histogram') {
+      return {
+        combine: true,
```

Comment:
i don't think we want this. unless the old panel combined multiple series into one. iiuc the old one simple has a reducer per series and can optionally stack the result of that.

Reply:
actually nvm, i think my brain is in `series` / BarChart mode. but still, this should only enabled when stacking is off in the old panel?